Lib/pathlib.py

```diff
@@ -23,6 +24,13 @@
 # Internals
 #
 
+_WIN_DRIVE_LETTERS = frozenset(string.ascii_letters)
+_WIN_RESERVED_NAMES = (
```

Comment:
Any reason for this not to be a frozenset too?

```suggestion
_WIN_RESERVED_NAMES = frozenset(
```